typescript/zetachain/zetacore/observer/ballot_pb.ts

```diff
@@ -13,19 +13,14 @@ import type { Message } from "@bufbuild/protobuf";
  * Describes the file zetachain/zetacore/observer/ballot.proto.
  */
 export const file_zetachain_zetacore_observer_ballot: GenFile = /*@__PURE__*/
-  fileDesc("Cih6ZXRhY2hhaW4vemV0YWNvcmUvb2JzZXJ2ZXIvYmFsbG90LnByb3RvEht6ZXRhY2hhaW4uemV0YWNvcmUub2JzZXJ2ZXIi5QIKBkJhbGxvdBINCgVpbmRleBgBIAEoCRIZChFiYWxsb3RfaWRlbnRpZmllchgCIAEoCRISCgp2b3Rlcl9saXN0GAMgAygJEjQKBXZvdGVzGAQgAygOMiUuemV0YWNoYWluLnpldGFjb3JlLm9ic2VydmVyLlZvdGVUeXBlEkYKEG9ic2VydmF0aW9uX3R5cGUYBSABKA4yLC56ZXRhY2hhaW4uemV0YWNvcmUub2JzZXJ2ZXIuT2JzZXJ2YXRpb25UeXBlEj0KEGJhbGxvdF90aHJlc2hvbGQYBiABKAlCI8jeHwDa3h8bY29zbW9zc2RrLmlvL21hdGguTGVnYWN5RGVjEkAKDWJhbGxvdF9zdGF0dXMYByABKA4yKS56ZXRhY2hhaW4uemV0YWNvcmUub2JzZXJ2ZXIuQmFsbG90U3RhdHVzEh4KFmJhbGxvdF9jcmVhdGlvbl9oZWlnaHQYCCABKAMiQQoTQmFsbG90TGlzdEZvckhlaWdodBIOCgZoZWlnaHQYASABKAMSGgoSYmFsbG90c19pbmRleF9saXN0GAIgAygJKlEKCFZvdGVUeXBlEhYKElN1Y2Nlc3NPYnNlcnZhdGlvbhAAEhYKEkZhaWx1cmVPYnNlcnZhdGlvbhABEg8KC05vdFlldFZvdGVkEAIaBKikHgEqegoMQmFsbG90U3RhdHVzEiYKIkJhbGxvdEZpbmFsaXplZF9TdWNjZXNzT2JzZXJ2YXRpb24QABImCiJCYWxsb3RGaW5hbGl6ZWRfRmFpbHVyZU9ic2VydmF0aW9uEAESFAoQQmFsbG90SW5Qcm9ncmVzcxACGgSopB4BQukBCh9jb20uemV0YWNoYWluLnpldGFjb3JlLm9ic2VydmVyQgtCYWxsb3RQcm90b1ABWitnaXRodWIuY29tL3pldGEtY2hhaW4vbm9kZS94L29ic2VydmVyL3R5cGVzogIDWlpPqgIbWmV0YWNoYWluLlpldGFjb3JlLk9ic2VydmVyygIbWmV0YWNoYWluXFpldGFjb3JlXE9ic2VydmVy4gInWmV0YWNoYWluXFpldGFjb3JlXE9ic2VydmVyXEdQQk1ldGFkYXRh6gIdWmV0YWNoYWluOjpaZXRhY29yZTo6T2JzZXJ2ZXJiBnByb3RvMw", [file_gogoproto_gogo, file_zetachain_zetacore_observer_observer]);
```

Comment:
not related but maybe we can consider adding typescript folder to gitattributes so this diff in future is not showing

Reply:
Yeah I would not consider it as a big issue for now